### DISCUSSION TITLE
This plugin is awesome! - Just positive feedback

### DISCUSSION THREAD

**Discussion by Nerocraft4** [Vinzent03/obsidian-git#479] [General]
No need for the devs to reply, but man, this is awesome! I can't emphasize it enough, it's really cool. Now I can work with my phone and my computer in the same project, and even collaborate with my colleagues. It is really a life saver.
By the way, was wondering if the pull-push notifications can be deactivated? I have the back-up frequency set to like one minute, and they take up a decent amount of space in my phone screen. Thank you in advance!

**Comment by Vinzent03** [comment]
Happy you like it! There is a toggle in the plugin settings to disable notifications.
